ccd-definition/CaseTypeTab.json

```diff
@@ -48,17 +48,8 @@
     "TabLabel": "Draft orders",
     "TabDisplayOrder": 3,
     "CaseFieldID": "caseManagementOrder",
-    "TabFieldDisplayOrder": 1
-  },
-  {
-    "LiveFrom": "01/01/2017",
-    "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "Channel": "CaseWorker",
-    "TabID": "DraftOrdersCmoTab",
-    "TabLabel": "Draft orders",
-    "TabDisplayOrder": 3,
-    "CaseFieldID": "schedule",
-    "TabFieldDisplayOrder": 2
+    "TabFieldDisplayOrder": 1,
+    "Comment": "Is this supposed to be in DraftOrdersCmoTab or DraftOrdersTab"
```

Comment:
I guess we could reuse DraftOrdersTab? no reason to have specific CMO tab...

Reply:
That was my thoughts, just double checking